packages/zosfiles/src/api/methods/copy/Copy.ts

```diff
@@ -62,6 +62,10 @@ export class Copy {
             },
         };
 
+        if (options.enq !== undefined) {
+            payload.enq = options.enq;
+        }
```

Comment:
Before we continue to add more, I'd suggest enhancing the ICopyDataSetOptions API and just use that object as the payload :)

Reply:
Hi @zFernand0 , I have a new commit. Is that what you mean? I'm not sure if I get you correctly.